atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -91,6 +91,11 @@ private static void logTimeAndTableCount(String method, int tableCount, Stopwatc
                 method, tableCount, stopwatch.elapsed(TimeUnit.MILLISECONDS));
     }
 
+    private static void logTimeAndTableRange(String method, String tableName, RangeRequest range, Stopwatch stopwatch) {
```

Comment:
mine as well use this to update the other range request running operations in here to have them log the range as well (e.g. getRAngeOfTimestamps)